src/main/resources/application.properties

```diff
@@ -3,7 +3,7 @@ spring.datasource.username=postgres
 spring.datasource.password=mysecretpassword
 spring.datasource.hikari.maximumPoolSize=5
 
-server.port=8080
+server.port=8084
```

Comment:
will this port change affect anything in aws?

Reply:
no, but when u test locally, with the admin running on 8080, and user service on 8082, 8084 was the natural call for it 😅